CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## main / unreleased
+* [ENHANCEMENT] Query-scheduler: Introduce `query-scheduler.prioritize-query-components`, which allows configuration of the request queue tree to prioritize dequeuing from a specific query component more highly than dequeueing from a specific tenant. #7873
```

Comment:
do we also need to mention the `use_multi_algorithm_query_queue` here?

Reply:
Oh, I think this was a rebase oops -- prioritize-query-components is not available yet, only `use-multi-algorithm-query-queue` should be available